content/docs/views-and-templates/inertia.md

```diff
@@ -518,9 +518,11 @@ import type { HttpContext } from '@adonisjs/core/http'
 import type { NextFn } from '@adonisjs/core/types/http'
 
 export default class MyMiddleware {
-  public async handle({ inertia }: HttpContext, next: NextFn) {
-    inertia.share('appName', 'My App')
-    inertia.share('user', (ctx) => ctx.auth?.user)
+  public async handle({ inertia, auth }: HttpContext, next: NextFn) {
```

Comment:
We don't use `public` modifiers anywhere. So it will be nice to keep docs uniformed :)

Reply:
I did that already in this commit :) https://github.com/adonisjs/v6-docs/commit/33a9cfc153b0514fe16d4ae19e1ba7ef742a1746